packages/web-pkg/src/composables/capability/useCapability.ts

```diff
@@ -56,10 +56,25 @@ export const useCapabilityProjectSpacesEnabled = createCapabilityComposable(
 export const useCapabilityShareJailEnabled = createCapabilityComposable('spaces.share_jail', false)
 
 export const useCapabilityReadOnlyUserAttributes = createCapabilityComposable(
-  'graph.read_only_user_attributes',
+  'graph.users.read_only_attributes',
   []
 )
 
+export const useCapabilityCreateUsersDisabled = createCapabilityComposable(
+  'graph.users.create_disabled',
+  false
+)
+
+export const useCapabilityDeleteUsersDisabled = createCapabilityComposable(
+  'graph.users.delete_disabled',
+  false
+)
+
+export const useCapabilityChangeSelfPasswordDisabled = createCapabilityComposable(
+  'graph.users.change_password_self_disabled',
+  false
```

Comment:
```suggestion
  true
```

this fixes the oc10 weirdness and fine to set it like this for the product. ocis announces the capability. You only need the fallback for backends that don't have the capability (which is only oc10).

Reply:
should I apply true to this capability or the others as well (would favor only apply it to this capability)